src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -449,12 +449,11 @@ def dtype(self) -> DtypeObj:
         return self.to_pandas().dtype
 
     @property
-    @index_not_implemented()
-    def shape(self) -> None:
+    def shape(self) -> tuple:
```

Comment:
Add sections for what is returned and examples for each API you implemented here.